src/Softeq.NetKit.Chat.Data.Persistent.Sql/Repositories/MemberRepository.cs

```diff
@@ -28,18 +28,33 @@ public async Task<QueryResult<Member>> GetPagedMembersAsync(int pageNumber, int
         {
             using (var connection = _sqlConnectionFactory.CreateConnection())
             {
-                var whereCondition = !string.IsNullOrEmpty(nameFilter) ? " WHERE LOWER(Name) LIKE LOWER('%' + @nameFilter + '%')" : "";
-
-                var sqlQuery = @"SELECT *
-                                 FROM Members " +
-                                 whereCondition +
-                               @" ORDER BY Name
-                                 OFFSET @pageSize * (@pageNumber - 1) ROWS
-                                 FETCH NEXT @pageSize ROWS ONLY
-
-                                 SELECT COUNT(*)
-                                 FROM Members" +
-                                 whereCondition;
+                var sqlQuery = $@"
+                    SELECT
+                        {nameof(Member.Id)},
+                        {nameof(Member.Email)},
+                        {nameof(Member.IsAfk)},
+                        {nameof(Member.IsBanned)},
+                        {nameof(Member.LastActivity)},
+                        {nameof(Member.LastNudged)},
+                        {nameof(Member.Name)},
+                        {nameof(Member.PhotoName)},
+                        {nameof(Member.Role)},
+                        {nameof(Member.SaasUserId)},
+                        {nameof(Member.Status)},
+                        {nameof(Member.IsActive)}
+                    FROM 
```

Comment:
When we'll merge pull request with new chat version, we need to be sure that we'll add new property:
m.{nameof(Member.IsDeleted)}

Reply:
We won't forget :)